test/replicate.test.js

```diff
@@ -4,7 +4,6 @@ const Keys = require('ssb-keys')
 const { Server, replicate } = require('./helpers')
 
```

Comment:
Really stoked to have this in.
There's some comment and file tidying to be done but other than those I think this is good to go